opentelemetry-integration-example/README.rst

```diff
@@ -0,0 +1,12 @@
+OpenTelemetry Integration Example
```

Comment:
The term "Integration" is already in use for our "ext" packages. Maybe "Example application monitored with OpenTelmetry"?

Reply:
good point. can do. How about the package name is that "opentelemetry-example"?